Sources/StreamChatUI/Utils/AudioPlayer/AudioPlaybackRate.swift

```diff
@@ -0,0 +1,57 @@
+//
+// Copyright © 2023 Stream.io Inc. All rights reserved.
+//
+
+import Foundation
+
+// Defines an enumeration for audio playback rate with a raw value of Float
+public enum AudioPlaybackRate: Float, CustomStringConvertible {
```

Comment:
Not sure if an enum is the best representation here - we would need to always add additional cases, for example if customers ask for 1.25 or 1.5 (the more common ones).